sentinel-core/src/main/java/com/alibaba/csp/sentinel/log/jul/ConsoleHandler.java

```diff
@@ -43,9 +50,21 @@ class ConsoleHandler extends Handler {
      */
     private StreamHandler stderrHandler;
 
+    private ExecutorService executor;
+
     public ConsoleHandler() {
         this.stdoutHandler = new StreamHandler(System.out, new CspFormatter());
         this.stderrHandler = new StreamHandler(System.err, new CspFormatter());
+
+        int corePoolSize = 1;
+        int maximumPoolSize = 1;
+        long keepAliveTime = 0;
+        /**insure the log can be recorded*/
+        int queueSize = 1024;
+        RejectedExecutionHandler handler = new ThreadPoolExecutor.DiscardPolicy();
```

Comment:
DiscardPolicy may prevent us from knowing if any logs were discarded, which can be misleading when troubleshooting. FYR, using the AbortPolicy and adding exception handling may be a way to avoid this kind of misleading.

Reply:
defined RejectedExecutionHandler to record the rejected log task
1.AbortPolicy would stop record tasks, may be not the better choice
2.another problem is that it's not easy to record the message when log task is rejected due to the task itself is to log something, so just prints the rejected record

Reply:
Looks cool! FYR, we may not need to print full content though System.err. As prompting once when the log is discarded for the first time in a certain period of time may be enough to avoid misleading. BTW, the label of sentinel when print through System.err should be needed, in order to let the user know that the output comes from Sentinel.

Reply:
has add sentinel tag to identify the record log is printed by sentinel SDK.

Reply:
LGTM